examples/views/button_roles.py

```diff
@@ -26,11 +23,12 @@ def __init__(self, role: discord.Role):
                          style=discord.enums.ButtonStyle.primary, custom_id=str(role.id))
 
     async def callback(self, interaction: discord.Interaction):
-        """This function will be called any time a user clicks on this button
+        """
+        This function will be called any time a user clicks on this button
 
         Parameters
         ----------
-        interaction : discord.Interaction
+        interaction: discord.Interaction
             The interaction object that was created when the user clicked on the button
```

Comment:
Could we just remove this from the docstring? Doesn't feel like we need docs for examples.